api-server/api-server-common/src/storage/impls/postgres/queries.rs

```diff
@@ -2149,6 +2427,15 @@ impl<'a, 'b> QueryFromConnection<'a, 'b> {
             .await
             .map_err(|e| ApiServerStorageError::LowLevelStorageError(e.to_string()))?;
 
+        let coin_or_token_id = CoinOrTokenId::TokenId(token_id);
+        self.tx
+            .execute(
+                "INSERT INTO ml.coin_or_token_decimals (coin_or_token_id, block_height, number_of_decimals) VALUES ($1, $2, $3);",
+                &[&coin_or_token_id.encode(), &height, &0_i16],
+            )
+            .await
+            .map_err(|e| ApiServerStorageError::LowLevelStorageError(e.to_string()))?;
+
         Ok(())
```

Comment:
Same comment.

And btw probably the above "INSERT INTO ml.nft_issuance" should also have "ON CONFLICT ... UPDATE" so that it behaves just like its in-memory counterpart.

Reply:
Doesn't have an on conflict because it cannot happen same as the insert of the nft above it.
Added a panic in the in-memory counterpart.